sdk/appconfiguration/app-configuration/typedoc.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
/cc @KarishmaGhiya 
Does your script that generates docs look at typedoc.json files? Is there a way to specify project-specific config?

Reply:
Yes you can refer this example - https://github.com/Azure/azure-sdk-for-js/blob/feature/storage/sdk/storage/storage-file/typedoc.json

Reply:
Oh awesome, did not know that!

Reply:
:)

Reply:
@KarishmaGhiya  - that's for the link. I added in two missing attributes - name and readme.